Index: src/main/org/testng/internal/Invoker.java
===================================================================
--- src/main/org/testng/internal/Invoker.java	(revision 767)
+++ src/main/org/testng/internal/Invoker.java	(working copy)
@@ -482,6 +482,8 @@
       testResult.setStatus(ITestResult.STARTED);
       runTestListeners(testResult);
 
+      parameterValues = testResult.getParameters();
+      
       invokedMethod= new InvokedMethod(instances[instanceIndex],
           tm,
           parameterValues,
